css/components/page/home/_home.scss

```diff
@@ -0,0 +1,98 @@
+.l-home-header {
+  padding: 150px 0 80px 0;
+  background-image: url('/static/images/home-header.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  @media (min-width: $screen-m) {
+    padding: 320px 0 80px 0;
+  }
+}
+
+.l-home-intro {
+  padding-bottom: 50px;
+
+  @media (min-width: $screen-m) {
+    padding-bottom: 80px;
+  }
+
+  .video-wrapper {
+    position: relative;
+    padding-bottom: 56.25%;
+    padding-top: 30px;
+    height: 0;
+    overflow: hidden;
+    margin: 30px 0;
+
+    iframe {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+    }
+  }
+}
+
+.l-home-main-links {
+  padding-top: 50px;
+
+  @media (min-width: $screen-m) {
+    padding: 50px 0 80px 0;
+  }
+}
+
+.m-main-link {
```

Comment:
We don't use modules anymore. They are components. If you are going to use exclusively here, there's no need to put them in `components` folder.